docs/subsites.md

```diff
@@ -0,0 +1,68 @@
+---
+id: subsites
+title: Creating Subsites
+sidebar_label: Creating Subsites
+---
+
+## Creating a Subsite
+
+Subsites are versions of a main site that have a different host/path configuration and different content, but site application code is shared. This feature works great with locale and internationalization support. 
+
+There might be a time when you want to expand your site, whether regionally, categorically, or for a variety of other reasons. For instance, say you have a sports site. In the US, you'd want your site to use a .com TLD and have its content focused on the American sports of Baseball, Football, etc. Now, if you want to expand into the UK, you might want to use a .co.uk domain, and have content more geared toward Football (Soccer), Rugby, and Cricket. Subsites give you this ability, while allowing for shared logic in components, styles, etc.
+
+example component model.js:
```

Comment:
Let's make this a title to add more emphasis. This is just a suggestion, definitely no blocker at all
```suggestion
### Example
```